### ISSUE TITLE
Windows - GPT4ALL not open

### ISSUE THREAD

**Issue by rxqviz** [nomic-ai/gpt4all-chat#19]
https://user-images.githubusercontent.com/130754689/232023648-5ae39e10-f4a0-4008-ad0c-f44d9485556a.mp4

**Comment by elementalcollision** [comment]
Curious as to what you're trying to show here?

**Comment by gmy77** [comment]
Same here i click to CHAT on Windows 11 but nothing happens... nothing.

**Comment by manyoso** [comment]
Have heard of similar reports. Thanks for the video btw!!! 

Can you try and launch via command line? Also check if windows defender is causing problems... actively investigating this issue.

**Comment by gitonelove** [comment]
Windows 10,same problem.When I start via cmd nothing changes,the process starts for one second and then closes.

**Comment by gitonelove** [comment]
Xeon e3-1230v2,rx 580 8gb VRAM, 16gb RAM.

**Comment by gmy77** [comment]
`[-](https://github.com/nomic-ai/gpt4all-chat/issues/19#) <Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
[-](https://github.com/nomic-ai/gpt4all-chat/issues/19#) <System>
  <Provider Name="Application Error" Guid="{a0e9b465-b939-57d7-b27d-95d8e925ff57}" /> 
  <EventID>1005</EventID> 
  <Version>0</Version> 
  <Level>2</Level> 
  <Task>100</Task> 
  <Opcode>0</Opcode> 
  <Keywords>0x8000000000000000</Keywords> 
  <TimeCreated SystemTime="2023-04-14T11:39:24.3120225Z" /> 
  <EventRecordID>3623</EventRecordID> 
  <Correlation /> 
  <Execution ProcessID="12068" ThreadID="3524" /> 
  <Channel>Application</Channel> 
  <Computer>Win11</Computer> 
  <Security UserID="S-1-5-21-3858718263-905062841-1618656775-1001" /> 
  </System>
[-](https://github.com/nomic-ai/gpt4all-chat/issues/19#) <EventData>
  <Data Name="FilePath" /> 
  <Data Name="AppName">chat.exe</Data> 
  <Data Name="StatusCode">0x0</Data> 
  <Data Name="MediumType">0x0</Data> 
  </EventData>
  </Event>`

This is Windows Application error reporting about "CHAT".

**Comment by rxqviz** [comment]
> Have heard of similar reports. Thanks for the video btw!!!
> 
> Can you try and launch via command line? Also check if windows defender is causing problems... actively investigating this issue.

I try this, its not work, i idk why. And Windows Defender off.

**Comment by gmy77** [comment]
> > Have heard of similar reports. Thanks for the video btw!!!
> > Can you try and launch via command line? Also check if windows defender is causing problems... actively investigating this issue.
> 
> I try this, its not work, i idk why. And Windows Defender off.

I try various scenarios... and nothing.

**Comment by chuSS** [comment]
The same. Exit without any errors.
Win11. Xeon E5-1650 v2. RAM 32Gb. GTX 1060 3GB.

**Comment by andzejsp** [comment]
same here, launching chat.exe does not launch
Could it be that i dont have avx2 support?
![image](https://user-images.githubusercontent.com/80409979/232048149-f1c34340-b782-48ee-8e22-d1c985c4a218.png)

**Comment by gmy77** [comment]
> same here, launching chat.exe does not launch Could it be that i dont have avx2 support? ![image](https://user-images.githubusercontent.com/80409979/232048149-f1c34340-b782-48ee-8e22-d1c985c4a218.png)

I noted other people have the same problem but have avx2...

**Comment by LongBeardMan** [comment]
I am also having this issue. I open the chat.exe via desktop link and command line and nothing appears (even running as admin). What info is needed to help debug this? Hopefully I can help in some way

**Comment by manyoso** [comment]
I wonder if it is a problem with Xeon/avx2 support or similar... everyone who has this problem can you report your hardware chipset and whether you support avx2 or not?

**Comment by gmy77** [comment]
> I wonder if it is a problem with Xeon/avx2 support or similar... everyone who has this problem can you report your hardware chipset and whether you support avx2 or not?

Intel(R) Core(TM) i5 CPU         750  @ 2.67GHz   2.67 GHz

**Comment by LongBeardMan** [comment]
> I wonder if it is a problem with Xeon/avx2 support or similar... everyone who has this problem can you report your hardware chipset and whether you support avx2 or not?

I have Intel Core i7-2760QM and it does not support AVX2 according to hwinfo64. So you might be on to something

**Comment by gitonelove** [comment]
My processor does not have avx2, but another program (I do not remember its exact name, sorry), works great without avx2. in general, I wonder if it is possible to run a neural network using the video card and the processor?

**Comment by andzejsp** [comment]
devs just need to add a flag to check for avx2, and then when building pyllamacpp 
https://github.com/nomic-ai/gpt4all-ui/issues/74#issuecomment-1508365776

Given that this is related. 

I did built the pyllamacpp this way but i cant convert the model, because some converter is missing or was updated and the gpt4all-ui install script is not working as it used to be few days ago.

**Comment by gitonelove** [comment]
> devs just need to add a flag to check for avx2, and then when building pyllamacpp [nomic-ai/gpt4all-ui#74 (comment)](https://github.com/nomic-ai/gpt4all-ui/issues/74#issuecomment-1508365776)
> 
> Given that this is related.
> 
> I did built the pyllamacpp this way but i cant convert the model, because some converter is missing or was updated and the gpt4all-ui install script is not working as it used to be few days ago.

I take it that I have nothing to do here without the avx2? :DD

**Comment by andzejsp** [comment]
It works without avx2 allegedly, on other private repos people have tested etc. Give devs some time i guess.. 
If you know a solution, please share :)

**Comment by gmy77** [comment]
--seed
--threads
--prompt
--n_predict
--top_k
--top_p
--temp
--batch_size
--model
--help

Unfortunately entering the available options on the command line doesn't work either.

**Comment by gmy77** [comment]
> It works without avx2 allegedly, on other private repos people have tested etc. Give devs some time i guess.. If you know a solution, please share :)

Yes i search inside and if i view correctly they added AVX2 and AVX1, but for my personal very limited knowledge problem reside at start.

**Comment by manyoso** [comment]
This is related... https://github.com/ggerganov/ggml/issues/86 I could turn off avx2 but then people who have it will be screwed with worse performance. The key is to have a number of builds each supporting different os/arch/intrinsics

**Comment by biraescudero** [comment]
I have the same problem. Nothing happens when I try to execute on Windows. Fix it woud be great. Hope you find the bug,

**Comment by flimblet** [comment]
im having the same problem. when i try to execute the chat.exe, nothing happens.

**Comment by Aliync** [comment]
well, ii have the same problem opening GPT4All and nothing...

**Comment by Rubinsergei** [comment]
Hi all. Any new ideas for a startup with this problem? I also encountered it

**Comment by sakh-01** [comment]
https://www.intel.com/content/www/us/en/download/684897/684910/intel-software-development-emulator.html
I was able to run it with the avx2 emulator, but it is very slow

**Comment by pixezy** [comment]
So it looks like it will be impossible to use GPT4All on my i3 processor...

**Comment by Sakhalin-01** [comment]
Waiting to hear back from [manyoso]

**Comment by gmy77** [comment]
> https://www.intel.com/content/www/us/en/download/684897/684910/intel-software-development-emulator.html I was able to run it with the avx2 emulator, but it is very slow

Slowly but yes work...

**Comment by invdj** [comment]
Work in Windows 10 with Intel SDE. 
![gpt4all-windows](https://user-images.githubusercontent.com/37749913/232326948-19648238-d438-4cdb-b977-c645f067255f.JPG)
Will be good devs put system requirements somewhere.

**Comment by HappyPony** [comment]
@gmy77 
> das "hsw" ist der Prozessortyp, aus der Liste "-help" alle Kommandozeilen-Admins auswählen.

Can you please show me how to determine the value "hsw"?

**Comment by manyoso** [comment]
Please try again with latest release.

**Comment by fabiodirauso** [comment]
the problem persistent.

**Comment by HappyPony** [comment]
@fabiodirauso To which version does your statement 
> the problem persistent.

refer?
